website/graphql/Connections.md

```diff
@@ -166,7 +166,11 @@ certain optimizations if this field returns an object that implements
 ### Cursor
 
 An "Edge Type" must contain a field called `cursor`. This field must return
-either a String or a Non-Null wrapper around a String.
+either a type that serializes as a String; this may be a String, a Non-Null
+wrapper around a String, or a custom scalar that serializes as a String.
```

Comment:
I think it might not be clear from this wording whether Non-Null wrapper is allowed for a custom scalar, but I believe this is the intent anyway? I.e. it may be a String, a custom scalar that serializes as a String or a Non-Null wrapper around either.